native/checkout-to-tag.sh

```diff
@@ -8,7 +8,7 @@ export DEPOT_TOOLS_UPDATE=0
 CHROMIUM_SRC_ROOT=${CHROMIUM_SRC_ROOT:-/root/chromium/src}
 DEPOT_TOOLS_ROOT=${DEPOT_TOOLS_ROOT:-/root/depot_tools}
 export PATH="$DEPOT_TOOLS_ROOT:$PATH"
-TAG=${1:-108.0.5359.243}
+TAG=${1:-128.0.6537.2}
```

Comment:
looking back through some conversations on element in october/november about finding a working cronet build, and also one of your earlier commits (https://github.com/greatfire/envoy/pull/98/commits/ff53e3cae2e72904914f57fdff3d4a8713ead800) i thought the version that worked was 128.0.6557.2

Reply:
`128.0.6537.2` is the correct version, `128.0.6557.2` was a typo I made at some point (the double 5 is the clue), I've done my best to purge everything that said `128.0.6557.2`